### ISSUE TITLE
[Help] About the hyper-parameters to reproduce the result

### ISSUE THREAD

**Issue by yeeeqichen** [snap-stanford/GreaseLM#6]
@XikunZhang @michiyasunaga @roks 

Hi,

Thanks for your great effort! 

I've run the code in this repo with the same hyper-parameters provided in the script `run_greaselm.sh`, which are also the same as reported in the paper. But the results aren't as good as reported in the paper. For example, in csqa, the reported `dev_acc` and `test_acc` are 78.5(+-0.5) and 74.2(+-0.4) respectively, but the model I trained only performs 77.48 and 73.01 respectively.

I've tried several random seeds, but the problem still exists. So could you please release the hyper-parameters(i.e. random seed) that you used when you train the model?

Look forward to your response!

**Comment by XikunZhang** [comment]
Thanks for your interest in our work! I am sorry to hear about your difficulty in reproducing the results.

Unfortunately, the randomness is not fully controlled in our code. The only sources of randomness we control are [these](https://github.com/snap-stanford/GreaseLM/blob/803946bba3273556c1ff2be6ad8b02850fe5972d/greaselm.py#L40-L44), but PyTorch still has other [sources of randomness](https://pytorch.org/docs/stable/notes/randomness.html), which means that even if you use the same random seeds as us, you still won't get the exact same numbers. Also from my personal experience, using different types of GPUs is another source of randomness.

Lmk if we can be further helpful in any kind.

**Comment by yeeeqichen** [comment]
Thanks for your response!
